Project.toml

```diff
@@ -5,6 +5,7 @@ version = "1.3.5"
 
 [deps]
 Extents = "411431e0-e8b7-467b-b5e0-f676ba4f2910"
+InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
```

Comment:
@rafaqz We could also directly depend on GeoFormatTypes? To really integrate them (some of our stuff should return a GeoFormatType).